graphics/opencv4/Portfile

```diff
@@ -19,13 +19,13 @@ maintainers         \
 
 description         Open Source Computer Vision Library
 
-long_description    OpenCV (Open Source Computer Vision Library) is an\
-                    open source computer vision and machine learning\
-                    software library. OpenCV was built to provide a\
-                    common infrastructure for computer vision\
-                    applications and to accelerate the use of machine\
-                    perception in the commercial products. Being a\
-                    BSD-licensed product, OpenCV makes it easy for\
+long_description    OpenCV (Open Source Computer Vision Library) is an \
```

Comment:
I feel like I'm being nit-picky here - and that's not my intent - but you don't want spaces between the last word on a description line, and the backslash. Otherwise you end up with two spaces in a row, where these breaks occur.

That also goes for things like port notes too.

Reply:
Scratch that, this only applies to portfile notes, apparently. Never mind! Lol